daemon/call.c

```diff
@@ -477,6 +495,7 @@ static void call_timer(void *ptr) {
 	struct rtp_stats *rs;
 	unsigned int pt;
 	endpoint_t ep;
+	int offers, answers, deletes;
```

Comment:
These should probably be of type u_int64_t if you don't want to risk overflows.

Reply:
I modified this, as this can become an issue if there are many calls